README.md

```diff
@@ -1,3 +1,5 @@
+Recommended alternative for PostHog cloud users: S3 batch exports
+
 # PostHog Databricks Plugin
 
```

Comment:
The content is fine, but it reads a bit ambiguous: what is the alternative to what? Maybe, we can be a bit more explicit and eye-catching:
```suggestion
# PostHog Databricks Plugin

> ℹ️ This plugin is only available for self-hosted users. Users of PostHog cloud should look into [S3 batch exports](https://posthog.com/docs/cdp/batch-exports/s3) and [connecting databricks to S3](https://docs.databricks.com/en/storage/amazon-s3.html) as an alternative.
```

Reply:
A bit nitty though, so feel free to ignore. I'll approve the PR.

Reply:
thank you :)